animalai/raycastparser.py

```diff
@@ -28,6 +28,7 @@ class RayCastObjects(enum.Enum):
     SIGNPOSTER = 11
     DECAYGOAL = 12
     DECAYGOALBOUNCE = 13
+    HOLLOWBOX = 14
```

Comment:
Added this with this PR as otherwise it would have been delayed until the next major PR for python codebase. Standalone mod.

Reply:
I don't think this is best practice.